lib/model/store.dart

```diff
@@ -341,6 +342,54 @@ class PerAccountStore extends ChangeNotifier with ChannelStore, MessageStore {
 
   final Map<int, User> users;
 
+  Map<String, Wildcard> wildcardsForNarrow(Narrow narrow) => Map.fromEntries(
+    _wildcardsForNarrow(narrow).map((w) => MapEntry(w.name, w)));
+
+  List<Wildcard> _wildcardsForNarrow(Narrow narrow) {
```

Comment:
This should live in an autocomplete-specific file — `store.dart`, and the `PerAccountStore` class definition, are fairly crowded places, so anything that can naturally go somewhere more specific should do so.